### DISCUSSION TITLE
Object literal may only specify known properties, and 'x' does not exist in type

### DISCUSSION THREAD

**Discussion by MohamadrezaKazemian** [prisma/prisma#16300] [Q&A] [answered]
0

recently I've tried to add two new items to one of my tables in my Prisma. schema. after I update my database ( using Prisma DB push) and try to .create new data for my table I face some issue after I update my database ( using Prisma DB push) and try to .create new data for my table I face some problems. on my localhost the project is fine and all the APIs are working correctly. but when I deploy the project on the server ( using vercel ) it throws me an error :

`<html>
<body>

Type error: Type '{ businessCategory: any; website: any; businessIntroducing: any; businessTitle: any; capitalNeed: any; state: any; city: any; supplyChannels: any; supplyType: any; customerSection: any; generatedAt: string; ... 11 more ...; owner: { ...; }; }' is not assignable to type '(Without<businessesCreateInput, businessesUncheckedCreateInput> & businessesUncheckedCreateInput) \| (Without<...> & businessesCreateInput)'.
--
17:24:52.392 | Object literal may only specify known properties, and 'revenueModelsDescription' does not exist in type 'Without<businessesUncheckedCreateInput, businessesCreateInput> & businessesCreateInput'.
17:24:52.392 |  
17:24:52.392 | 41 \|                 relatedIndustry: formData.relatedIndustry,
17:24:52.392 | 42 \|                 revenueModels: formData.revenueModels,
17:24:52.392 | > 43 \|                 revenueModelsDescription : formData.revenueModelsDescription,
17:24:52.393 | \|                 ^
17:24:52.393 | 44 \|                 owner: { connect: { id: formData.ownerId } }
17:24:52.393 | 45 \|             }
17:24:52.393 | 46 \|         })

</body>
</html>`

p.s : im using next.js + prisma

**Comment by nurul3101** [comment]
Hey @MohamadrezaKazemian 👋 
Is `npx prisma generate` command a part of your build script?
You can have a look at this [build script](https://github.com/prisma/deployment-example-vercel/blob/main/package.json#L13) as a reference.

Was the revenueModelsDescription field added recently?

**Reply by MohamadrezaKazemian** [reply]
yes, I added revenueModelsDescription recently.

after I saw your answer I added `npx prisma generate` and now I'm facing this error : 
`<html>
<body>


Error: P3005
--
18:01:21.525 |  
18:01:21.525 | The database schema is not empty. Read more about how to baseline an existing production database: https://pris.ly/d/migrate-baseline
18:01:21.525 |  
18:01:21.552 | Error: Command "npm run vercel-build" exited with 1


</body>
</html>`

**Reply by nurul3101** [reply]
What are the commands in your build script?

**Reply by nurul3101** [reply]
@MohamadrezaKazemian Let me know if you are still facing the issue.

**Comment by MohamadrezaKazemian** [comment, marked as answer]
thank you for your help.
I fixed it by adding some script to my package.json file :

  "scripts": {
    "dev": "next dev",
    "build": "next build",
    "start": "next start",
    "vercel-build": "prisma generate && prisma db push && next build",
    "prisma:generate": "prisma generate"
  },
p.s: you can use migrate deploy instead of prisma db push

**Comment by ludralph** [comment]
Hi there,

To keep our discussions organized and focused on the most relevant topics, we’re reviewing and tidying up our backlog. As part of this process, we’re closing discussions that have already been marked as answered but remain open.

If this discussion still requires further input or clarification, feel free to reopen it or start a new one with updated details. Your contributions are invaluable to the community, and we’re here to help!

For more details about our priorities and vision for the future of Prisma ORM, check out our latest blog post: https://www.prisma.io/blog/prisma-orm-manifesto.

Thank you for your understanding and ongoing support of the Prisma community!
